src/Application/test/CalculatorClassGTest.cpp

```diff
@@ -15,8 +14,14 @@
  * @brief header files included 
  * 
  */
-#include "CalculatorApplcation.h"
+#include "CalculatorApplicationFactory.h"
+#include "DivideCalculator.h"
+#include "MultiplyCalculator.h"
+#include "SubtractCalculator.h"
+#include "AddCalculator.h"
```

Comment:
order of #includes should be in split by system and yours split by single empty line, then alphabetized.